### PR TITLE
Bugfixing

### LINKED ISSUE #1: Fix `devtools:check()` warnings and notes

### PR DESCRIPTION
**Pull request by sims1253**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -20,8 +20,9 @@ Suggests:
     rmutil,
     extraDistr,
     BBmisc,
+    RcppParallel,
```
Comment (sims1253) [inline DESCRIPTION]: What are using this for?

File: R/cloglognormal.R [hunk 2]
```diff
@@ -94,16 +94,13 @@ posterior_epred_cloglognormal <- function(prep) {
 #'
 #' @examples # Running the example might take a while and may make RStudio unresponsive.
 #' # Just relax and grab a cup of coffe or tea in the meantime.
-#' cloglog_data <- rcloglognormal(1000, 0.5, 2)
+#' raw_data <- rcloglognormal(1000, 0.5, 2)
```
Comment (sims1253) [inline R/cloglognormal.R]: you can just call it data

File: R/cloglognormal.R [hunk 3]
```diff
@@ -94,16 +94,13 @@ posterior_epred_cloglognormal <- function(prep) {
 #'
 #' @examples # Running the example might take a while and may make RStudio unresponsive.
```
Comment (sims1253) [inline R/cloglognormal.R]: Please remove these kinds of off-hand comments throughout the project. They are fun but add clutter to the documentation.

File: tests/testthat/test-lomax.R [hunk 4]
```diff
@@ -85,9 +85,9 @@ test_that("custom-lomax", {
 
   # Check of brms can fit the custom family and recover the intercept and shape
   expect_brms_family(
-    intercept = 5,
```
Comment (sims1253) [inline tests/testthat/test-lomax.R]: Why did you change this?

Comment (Yannick73) [reply tests/testthat/test-lomax.R]: Yeah, that change is probably not very sensible. It was intended to lower the risk of failure for this test.
I don't think, it makes a difference, so I will revert this change.

File: R/cloglognormal.R (hunk 2 above)
Comment (Yannick73) [reply R/cloglognormal.R]: Can do.

File: R/cloglognormal.R (hunk 3 above)
Comment (Yannick73) [reply R/cloglognormal.R]: I will shorten them. Two lines seem excessive. But beeing informed about this is important, so I would suggest, not to remove them completely.

Comment (sims1253) [reply R/cloglognormal.R]: I don't think they are necessary. They obviously fit a model which takes some time.

File: DESCRIPTION (hunk 1 above)
Comment (Yannick73) [reply DESCRIPTION]: RccpParallel was necassary in tests with the cmdstanr backend. Backend changed to rstan, so I will remove the package.

File: R/cloglognormal.R (hunk 3 above)
Comment (Yannick73) [reply R/cloglognormal.R]: That is correct. I thought to inform, because those examples will also make the RStudio console unresponsive. This behavior did seem odd to me, given I was unaware about the relation.
But I will remove them, as this behavior is not a big deal.

File: tests/testthat/test-lomax.R (hunk 4 above)
Comment (Yannick73) [reply tests/testthat/test-lomax.R]: I guess, Lomax brms is the one test, I observed failing the most often. If I had to guess: 1/5 times I hit devtools-test, the expect_brms of test-lomax fails.
Maybe we should think about relaxing this specific test somehow.

**Comment by Yannick73** [comment]
I corrected everything commented. Now the examples and doc are in good shape, imo. This branch should be ready for merge.
Only thing left open is, to think about the Lomax test.
